tools/install-deps-rpi.sh

```diff
@@ -0,0 +1,147 @@
+#!/bin/bash
```

Comment:
As I know, it is duplicated with ```target/raspberry-pi2_3/install-deps-raspberry-pi2_3.sh```. I suggest that you remove ```target/raspberry-pi2_3/install-deps-raspberry-pi2_3```.

Reply:
I leave it for the compatibility with the previous version. I just added the test system and I think the sub following should be done step by step.

Reply:
Okay, then let's remove the file with ```target/tegraTX1/install-deps-tegraTX1.sh```.